boards/stm32f3discovery/src/main.rs

```diff
@@ -607,6 +609,18 @@ pub unsafe fn reset_handler() {
     );
     stm32f303xc::adc::ADC1.set_client(adc);
 
+    let nonvolatile_storage = components::nonvolatile_storage::NonvolatileStorageComponent::new(
+        board_kernel,
+        &stm32f303xc::flash::FLASH,
+        0x08020000,
+        0x20000,
+        0x08000000,
+        0x20000,
```

Comment:
I think this should probably use the estorage / sstorage symbol approach the same way that Imix / nrf boards do

Reply:
Ok. I'll apply your suggestion.